src/wallet-account.ts

```diff
@@ -80,6 +80,24 @@ export class WalletConnection {
     _connectedAccount: ConnectedWalletAccount;
 
     constructor(near: Near, appKeyPrefix: string | null) {
+        if(typeof window === 'undefined') {
+            return new Proxy(this, {
```

Comment:
Should we update the JSDoc to include this new behavior?

Reply:
Great point 💯 will do 👍